csharp-basic-topics/RemoveIllegalCharsFromPathAndFilenames/RemoveIllegalCharsFromPathAndFilenames/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Text.RegularExpressions;
+
+namespace CodeMazeGuides.CSBasic
+{
+    public class Program
```

Comment:
Can we actually put ALL of the static methods into a separate class? I find it not looking good using `Program` in tests.